app/src/main/java/sdp/moneyrun/Riddle.java

```diff
@@ -33,6 +33,11 @@ public Riddle(String question, String correctAnswer, String firstAnswer, String
         this.fourthAnswer = fourthAnswer;
     }
 
+    /**
+     * Needed for the DataBase
+     */
+    public Riddle(){}
```

Comment:
Not sure, but I dont think this is necessary

Reply:
public argumentless constructor is needed for the Firebase database

Reply:
Exactly, riddles will be kept locally so I don't think there is a need for this.